frontend/src/components/Alert/Alert.tsx

```diff
@@ -40,7 +40,7 @@ const Alert = ({
 		<AntDesignAlert
 			{...props}
 			type={type}
-			className={classNames(props.className, styles.alert)}
+			className={clsx(props.className, styles.alert)}
```

Comment:
clicking alerts button at the top, then clicking to alert routes to `/1/alerts/alerts/433`

![image](https://user-images.githubusercontent.com/1351531/218530731-1b5907a3-8edf-4d23-8f8b-4b55addef52b.png)

Reply:
Great catch! Fixed in [5abdee4](https://github.com/highlight/highlight/pull/4203/commits/5abdee41c5a38dc4dfec6a15dfda402bc1f0a050)